### ISSUE TITLE
Cannot use `rename` with `DjangoModelFactory`

### ISSUE THREAD

**Issue by jamesbeith** [rbarrois/factory_boy#236]
Is it possible to use `rename` with a `DjangoModelFactory`? Example:

```
class StandardFactory(factory.django.DjangoModelFactory):
    class Meta:
        model = models.StandardModel
        rename = {'attributes_': 'attributes'}

    attributes_ = factory.Sequence(lambda n: "attributes%d" % n)
```

When I try I get a `TypeError` saying `rename` is unknown.

```
TypeError: 'class Meta' for <class 'package.StandardFactory'> got unknown attribute(s) rename
```

**Comment by rbarrois** [comment]
Indeed, this fails on the latest release, but not on master. Your test fails on travis because it expects to get a non-renamed argument, but the renaming does work.

I plan to do a release tomorrow, this should fix your issue.

**Comment by rbarrois** [comment]
I've just released 2.6.0 which should fix your problem; feel free to reopen the issue if the problem is still there!
